File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/ClientRequestSearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface ClientRequestSearchModel extends BaseSearchModel {
    requestCode: string;
    branchId: number | null;
    branchCode: string | '';

    representativeId: number | null;
    representativeCode: string | '';

    clientId: number | null;
    clientCode: string | '';

    requestDate: Date | null;
    requestTypeId: number | null;
    requestTypeDetailId: number | null;
    phone: string;
    priorityId: number | null;
    requestStatusId: number | null;
    isClosed: number | null;
    departmentId: number | null;
}

File: SofiForce.Models/Models/SearchModels/ClientRequestSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class ClientRequestSearchModel : BaseSearchModel
    {
        public string RequestCode { get; set; }
        public int? BranchId { get; set; }
        public int? RepresentativeId { get; set; }
        public int? ClientId { get; set; }
        public DateTime? RequestDate { get; set; }
        public int? RequestTypeId { get; set; }
        public int? RequestTypeDetailId { get; set; }
        public string Phone { get; set; }
        public int? PriorityId { get; set; }
        public int? RequestStatusId { get; set; }
        public int? IsClosed { get; set; }
        public int? DepartmentId { get; set; }

    }
}